workflows/raredisease.nf

```diff
@@ -316,14 +316,15 @@ workflow RAREDISEASE {
     //
     // Create chromosome bed and intervals for splitting and gathering operations
     //
-    SCATTER_GENOME (
-        ch_genome_dictionary,
-        ch_genome_fai,
-        ch_genome_fasta
-    )
-    .set { ch_scatter }
-
-    ch_scatter_split_intervals  = ch_scatter.split_intervals  ?: Channel.empty()
+    ch_scatter_split_intervals = Channel.empty()
+    if (!(params.skip_subworkflows && params.skip_subworkflows.split(',').contains('snv_annotation'))) {
+        SCATTER_GENOME (
+            ch_genome_dictionary,
+            ch_genome_fai,
+            ch_genome_fasta
+        ).split_intervals
```

Comment:
.out.split_intervals ?

Reply:
`.out` part is not necessary when its part of invocation. But if we want to access SCATTER_GENOME's output elsewhere, then we need to use `.out`.